testsuite/MDAnalysisTests/core/test_groups.py

```diff
@@ -723,15 +769,16 @@ def _test_union(a, b, c, d, e):
         assert_array_equal(a.union(a).ix, np.arange(1, 5))
         assert_(a.union(d), np.arange(1, 5))
 
-    @staticmethod
-    def _test_intersection(a, b, c, d, e):
+    def test_intersection(self, groups):
+        a, b, c, d, e = groups
         intersect_ab = a.intersection(b)
         assert_array_equal(intersect_ab.ix, np.arange(3, 5))
         assert_(a.intersection(b) == b.intersection(a))
         assert_equal(len(a.intersection(d)), 0)
 
-    @staticmethod
-    def _test_subtract(a, b, c, d, e):
+    @pytest.mark.xfail
+    def test_subtract(self, groups):
```

Comment:
These tests have not been activated before. @richardjgowers

Reply:
Also ping @jbarnoud

Reply:
I probably just forgot to call the method in the litany of other test methods. I guess from the `xfail` that the test fail?

Reply:
Yes it does.